src/pages/HomePage/index.tsx

```diff
@@ -1,23 +1,37 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 
+import useUserContext from '~context/UserContext/UserContext';
+
 import styles from './homePage.module.less';
 
-const HomePage = () => (
-    <div className={styles.container}>
-        <h1 className={styles.header}>Jeg er...</h1>
-        <div className={styles.linkContainer}>
-            <Link to={'/soknad'} className={`${styles.link} knapp`}>
-                Veileder
-            </Link>
-            <Link to={'/saksoversikt'} className={`${styles.link} knapp`}>
-                Saksbehandler
-            </Link>
-            <Link to={'/attestering'} className={`${styles.link} knapp`}>
-                Attestant
-            </Link>
+const HomePage = () => {
+    const context = useUserContext();
+    return (
+        <div className={styles.container}>
+            <h1 className={styles.header}>Jeg er...</h1>
+            <div className={styles.linkContainer}>
+                <Link to={'/soknad'} className={`${styles.link} knapp`}>
+                    Veileder
+                </Link>
+                <Link to={'/saksoversikt'} className={`${styles.link} knapp`}>
+                    Saksbehandler
+                </Link>
+                <Link
+                    to={'/saksoversikt'}
```

Comment:
Det kommer ikke fra denne PR-en, men kunne vi ha brukt hjelpefunksjonene fra `Routes` for å lage URL-ene her?